# Dell PowerEdge servers seem to check "Fatal IO Error"s which triggers
# the script to exit with error message.
# Fix by ignoring the output of the 'sensor name' called "Fatal IO Error"
# tracked in https://bugzilla.novell.com/show_bug.cgi?id=836235

Index: check_ipmi_sensor_v1.3/check_ipmi_sensor
===================================================================
--- check_ipmi_sensor_v1.3.orig/check_ipmi_sensor
+++ check_ipmi_sensor_v1.3/check_ipmi_sensor
@@ -283,9 +283,12 @@ errorstring="Error"
 shopt -s nocasematch
 if [[ "$ipmioutput" =~ "${errorstring}" ]]
 then
+  if [[ "${errorstring}" =~ ^"Fatal IO Err" ]]
+  then
 	checkhint="check BMC availability/username/password"
 	echo "ipmitool output contains \"$errorstring\" - $checkhint"
 	exit 2
+  fi
 fi
 shopt -u nocasematch
 
